fbw-common/src/wasm/terronnd/build.sh

```diff
@@ -8,7 +8,7 @@ OUTPUT="${DIR}/out/terronnd.wasm"
 if [ "$1" == "--debug" ]; then
   CLANG_ARGS="-g"
 else
-  WASMLD_ARGS="--strip-debug"
+  WASMLD_ARGS="--strip-debug --gc-sections"
```

Comment:
Keep the common one.
```suggestion
  WASMLD_ARGS="--strip-debug"
```